tests/experimental/datasources/tasks.py

```diff
@@ -9,7 +9,14 @@
 import invoke
 import pandas as pd
 
-TEST_ROOT = pathlib.Path(__file__).parent.parent.parent.resolve(strict=True)
+from great_expectations.data_context.util import file_relative_path
+
+TEST_ROOT = pathlib.Path(
```

Comment:
Can you revert this back? One thing I really don't like in our codebase is the transformation of objects into strings and then back into objects again. If we want a Path object we should avoid turning it into a string unless that is needed by a third party lib. Alternatively you coud update `file_relative_path` to return a `Path` object but that is probably better done outside of this PR.

Reply:
@billdirks No, I should not modify `file_relative_path()` -- so I will revert the change -- that is more proper.  One moment.

Reply:
@billdirks This has been taken care of.  Thanks.